projects/emissary-ingress/emissary/helm/patches/0001-helm-changes.patch

```diff
@@ -4501,15 +4717,327 @@ index 2d06c8127..f1e043136 100644
+-    a8r.io/dependencies: {{ include "ambassador.fullname" . }}-redis.{{ include "ambassador.namespace" . }}
++    a8r.io/dependencies: {{ include "ambassador.fullname" . }}-redis.{{ .Release.Namespace | default .Values.defaultNamespace | quote }}
+ {{- if .Values.service.annotations }}
+   {{- range $key, $value := .Values.service.annotations }}
+-    {{ $key }}: {{ $value | quote }}
++    {{ $key }}: {{ $value |.Release.Namespace | default .Values.defaultNamespace | quote }}
+   {{- end }}
+ {{- end }}
+ spec:
+diff --git a/charts/emissary-ingress/templates/serviceaccount.yaml b/charts/emissary-ingress/templates/serviceaccount.yaml
+index ef30c7659..a20596210 100644
+--- a/charts/emissary-ingress/templates/serviceaccount.yaml
++++ b/charts/emissary-ingress/templates/serviceaccount.yaml
+@@ -3,7 +3,7 @@ apiVersion: v1
+ kind: ServiceAccount
+ metadata:
+   name: {{ include "ambassador.serviceAccountName" . }}
+-  namespace: {{ include "ambassador.namespace" . }}
++  namespace: {{ .Release.Namespace | default .Values.defaultNamespace | quote }}
+   labels:
+     app.kubernetes.io/name: {{ include "ambassador.name" . }}
+     {{- include "ambassador.labels" . | nindent 4 }}
+diff --git a/charts/emissary-ingress/templates/servicemonitor.yaml b/charts/emissary-ingress/templates/servicemonitor.yaml
+index b2c8122a1..2360b9031 100644
+--- a/charts/emissary-ingress/templates/servicemonitor.yaml
++++ b/charts/emissary-ingress/templates/servicemonitor.yaml
+@@ -3,7 +3,7 @@ apiVersion: monitoring.coreos.com/v1
+ kind: ServiceMonitor
+ metadata:
+   name: {{ include "ambassador.fullname" . }}
+-  namespace: {{ include "ambassador.namespace" . }}
++  namespace: {{ .Release.Namespace | default .Values.defaultNamespace | quote|.Release.Namespace | default .Values.defaultNamespace | quote }}
+   labels:
+     app: {{ include "ambassador.name" . }}
+     {{- if .Values.metrics.serviceMonitor.selector }}
+@@ -21,7 +21,7 @@ spec:
+       {{- end }}
+   namespaceSelector:
+     matchNames:
+-      - {{ include "ambassador.namespace" . }}
++      - {{ .Release.Namespace | default .Values.defaultNamespace | quote }}
+   selector:
+     matchLabels:
+       service: ambassador-admin
 diff --git a/charts/emissary-ingress/values.yaml.in b/charts/emissary-ingress/values.yaml
-similarity index 94%
+similarity index 93%
 rename from charts/emissary-ingress/values.yaml.in
 rename to charts/emissary-ingress/values.yaml
-index c2767e002..da2ee71f8 100644
+index c2767e002..f7e1c3b32 100644
 --- a/charts/emissary-ingress/values.yaml.in
 +++ b/charts/emissary-ingress/values.yaml
-@@ -6,7 +6,7 @@ fullnameOverride: ''
- namespaceOverride: ''
+@@ -2,11 +2,11 @@
+ nameOverride: ''
+ # Override the generated release name. Defaults to .Release.Name.
+ fullnameOverride: ''
+-# Override the generated release namespace. Defaults to .Release.Namespace.
+-namespaceOverride: ''
++
++defaultNamespace: emissary-system
  
  # Number of Ambassador replicas
 -replicaCount: 3
```

Comment:
Why  no replicaCount? (the diff of a patch file is confusing)

Reply:
Fixed.